README.md

```diff
@@ -1 +1,40 @@
-# Product page
\ No newline at end of file
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
I see that this is generic text. I would like to see more info about our project product-page so it becomes less generic